Plugins/NativeInput/Source/DeviceInputSystem.cpp

```diff
@@ -0,0 +1,82 @@
+#include "DeviceInputSystem.h"
+
+namespace Babylon::Plugins
```

Comment:
I think the convention we're establishing is that classes like this should live in the `Babylon::Plugins::Internal` namespace. @bghgary, please correct me if I'm mistaken about that.

https://github.com/BabylonJS/BabylonNative/blob/master/Plugins/NativeWindow/Source/NativeWindow.cpp#L3

Reply:
Well in this case, `DeviceInputSystem` is a private nested class of `Impl` which is a private nested class of `NativeInput`, so I can't change the namespace. These are just implementation details of `NativeInput`.